IdentityServer/v6/docs/content/tokens/pop/dpop.md

```diff
@@ -0,0 +1,93 @@
+---
+title: "DPoP"
+weight: 20
+---
+
+
+## Proof-of-possession using Demonstrating Proof-of-Possession at the Application Layer (DPoP)
+
+Added in 6.3.0.
+
+[DPoP](https://datatracker.ietf.org/doc/html/draft-ietf-oauth-dpop) specifies how to bind an asymmetric key stored within a JSON Web Key (JWK) to an access token. With this enabled your IdentityServer will embed the thumbprint of the public key JWK into the access token via the cnf claim, e.g.:
+
+```json
+{
+  // rest omitted
+  
+  "cnf": { "jkt": "JGSVlE73oKtQQI1dypYg8_JNat0xJjsQNyOI5oxaZf4" } 
+}
+```
+
+The client must then prove possession of the private key to call the APIs, and your APIs can [validate]({{< ref "/apis/aspnetcore/confirmation" >}}) the *cnf* claim by comparing it to the thumbprint of the client's public key in the JWK.
+
+If the access token would leak, it cannot be replayed without having access to the private key of the JWK the client controls.
+
+The mechanism by which the client proves control of the private key (both when connecting to the token server and when calling an API) is by sending an additional JWT called a proof token on HTTP requests.
+This proof token is passed via the *DPoP* request header and contains the public portion of the JWK, and is signed by the corresponding private key.
+
+The creation and management of this DPoP key is up to the policy of the client.
+For example is can be dynamically created when the client starts up, and can be periodically rotated.
+The main constraint is that it must be stored for as long as the client uses any access tokens (and possibly refresh tokens) that they are bound to.
+
+#### Enabling DPoP in IdentityServer
+
+DPoP is something a client can use dynamically with no configuration in IdentityServer, but you can configure it as required.
```

Comment:
I think this section needs a bit of clarification. I think what we're getting at is that you can always send a DPoP proof token to get a dpop token, but to require DPoP, you set the RequireDPoP flag? Is that right?

Reply:
And is there a configuration that would disable it entirely?

Reply:
The intent of the statement was to say that you don't need to mark the client as required for dpop. The client can just add the dpop request values without the IS config requiring it. IOW, the client can just do what the client(s) samples do, and DPoP will work. The IS config for require dpop just says that the client must use dpop.

Reply:
Ok, this doesn't need to hold up the pr, we can come back to it when/if we do the disable flag.